components/cli/pkg/commands/image/const.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
```

Comment:
issue in license header = http:www.wso2.org.

Reply:
Updated.